src/routes/users.js

```diff
@@ -4,14 +4,17 @@ import { createUser } from '../services/users.js';
 
 const userRouter = Router();
 
-userRouter.get('/', function (req, res, next) {
-  try {
-    res.send('users router');
-    createUser();
-  } catch (err) {
-    Logger.error('Error %o', err);
-    return next(err);
-  }
+userRouter.post('/', function (req, res, next) {
+  const body = req.body;
+  const user = {
+    email: body.email,
+    name: body.name,
+    pwd: body.pwd,
+    birthDate: body.birthDate,
+  };
+
+  createUser(user);
+  res.send('post');
```

Comment:
```suggestion
  res.status(200).json();
```

try-catch 문을 통해서 err가 잡히면 next(err)로 넘기는 작업이 필요할 것 같고, createUser도 무사히 저장되었는지 검사하여 저장되었다면 상태 코드 200과 함께 넘겨주는 등의 처리가 필요해보입니다 !!

Reply:
네 수정해서 다시 올릴게요!